pywincffi/kernel32/events.py

```diff
@@ -34,7 +34,8 @@ def CreateEvent(
         A pointer to a ``SECURITY_ATTRIBUTES`` structure.  If not provided
         then by default the handle cannot be inherited by a subprocess.
 
-    :keyword str lpName:
+    :keyword unicode/str lpName:
```

Comment:
I was going to comment on this and suggest:

```
:type lpName: unicode or str
:keyword lpName:
```
or 
```
:keyword str lpName:
```

After playing around with it some though, I don't think it matters (so this is more of an FYI I guess).  If you put just `str` in there sphinx will create a link to the standard string type.  It seems it allows `unicode/str` and the `:type ...` form I have above but neither of those create links back for the standard types.